### PR TITLE
Rochester guild: feat(tutorial): complete step 2

### PR DESCRIPTION
**Pull request by realModusOperandi**
I had to install @carbon/type to get the carbon--font-weight() function for some reason. Known issue?

### CODE REVIEW

File: src/content/LandingPage/LandingPage.js [hunk 1]
```diff
@@ -1,7 +1,102 @@
...
+                      its foundation, the system consists of working code,
+                      design tools and resources, human interface guidelines,
+                      and a vibrant community of contributors.
+                    </p>
+                    <Button>Learn more</Button>
+                  </div>
+                  <div className="bx--col-md-4 bx--offset-lg-1 bx--col-lg-8">
+                    <img
+                      className="landing-page__illo"
+                      src={'${process.env.PUBLIC_URL}/tab-illo.png'}
```
Comment (kellychurchill) [inline src/content/LandingPage/LandingPage.js]: Your single quotes should be backticks

**Comment by kellychurchill** [comment]
Yes - if you are starting fresh with step 2 which loads  "carbon-components": "^10.3.0", this is a problem introduced with the Carbon 10.3 release.

**Comment by kellychurchill** [comment]
👍 Looks good - @dstanich could you record completion of this step?

**Comment by dstanich** [comment]
Recorded the completion for step 2.

**Pull request closed without merging**